src/main/resources/XCarouselCode/XCarouselSheet.xml

```diff
@@ -890,15 +894,17 @@ colorPikerOn();</code>
     height: $carouselHeight;
     background-color: $carouselBGColor;
   }
+
   #if("$!{carouselObj.getProperty('carouselWidth').value}" == '')
     #set($carouselImageWidth = '100%')
   #else 
     #set($carouselImageWidth = ${carouselWidth})
   #end
-  $carouselID .carousel-inner>.item>img, $carouselID .carousel-inner>.item>a>img {
-    width: ${carouselImageWidth};
-    height: ${carouselHeight};
+  $carouselID .item-inner img {
+    max-width: ${carouselImageWidth};
```

Comment:
Firefox ignores max-width in certain situations, and in this case, if the image has a bigger width than the screen, it causes the image to not be fully displayed if the max-width is 100%.